video/vdu_sys.h

```diff
@@ -119,6 +125,35 @@ void VDUStreamProcessor::vdu_sys_video() {
 	auto mode = readByte_t();
 
 	switch (mode) {
+		case VDP_CURSOR_VSTART: {		// VDU 23, 0, &0A, offset
+			auto offset = readByte_t();	// Set the vertical start of the cursor
+			if (offset >= 0) {
+				cursorVStart = offset & 0x1F;
+				auto appearance = (offset & 0x60) >> 5;
```

Comment:
bits 5 and 6 of this byte controls the cursor appearance, specifically whether the cursor should be flashing, and the rate of flashing

behaviour here is as per RISC OS

The BBC Micro's Advanced User Guide describes the behaviour on the Beeb as using bit 6 enables or disables the blink feature, and bit 5 for the timing control bit.  this is very subtly different.  the RISC OS behaviour is close enough